.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: babel-loader
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint:
+    name: lint
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: yarn
+      - name: Lint
+        run: yarn run lint
+  test:
+    name: Test - ${{ matrix.os }} - Node v${{ matrix.node-version }}, Webpack ${{ matrix.webpack-version }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x, 14.x]
+        webpack-version: [latest, next]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: yarn
+      - name: Install webpack ${{ matrix.webpack-version }}
+        run: yarn add webpack@${{ matrix.webpack-version }}
+      - name: Run tests for webpack version ${{ matrix.webpack-version }}
+        run: yarn test-only
+      - name: Submit coverage data to codecov
```

Comment:
Can we submit coverage data only on ubuntu node 14.x webpack 4? So we don't submit multiple coverage data on a single PR: it is also more responsible since `coverage.io` is a free resource.

You can include additional variable such as `coverage: true` in the test matrix:
```yaml
include:
      - node: 14.x
        os: ubuntu-latest
        coverage: true
- uses: codecov/codecov-action@v1
   if: {{ matrix.coverage }}
```

Reply:
@JLHwung ok, anyway this PR never passes CI so I can't debug. can you review this PR at first? https://github.com/babel/babel-loader/pull/868